src/main/java/org/tdl/vireo/service/AssetService.java

```diff
@@ -87,7 +88,12 @@ public String find(String relativeFolderPath, String name) {
     }
 
     public void delete(String relativePath) throws IOException {
-        Files.delete(getAssetsAbsolutePath(relativePath));
+        Path path = getAssetsAbsolutePath(relativePath);
```

Comment:
Doesn't IOException handle file not found when it tries to read it? Should the controller advice log this warning?

Reply:
When reading file from IO to response stream the exception should occur.